File: src/main/resources/output/IFC4X3_vbnet/Entities/IfcExternalReference/IfcClassificationReference/IfcClassificationReference.vb
Public Class IfcClassificationReference Inherits IfcExternalReference
    Public ReferencedSource As IfcClassificationReferenceSelect
    Public Description As IfcText
    Public Sort As IfcIdentifier

    ' === INVERSE CLAUSES ===
    ' ClassificationRefForObjects : SET [0:?] OF IfcRelAssociatesClassification FOR RelatingClassification
    ' HasReferences : SET [0:?] OF IfcClassificationReference FOR ReferencedSource
End Class

File: src/main/resources/output/IFC4X3_csharp/Entities/IfcExternalReference/IfcClassificationReference/IfcClassificationReference.cs
public class IfcClassificationReference : IfcExternalReference
{
    public IfcClassificationReferenceSelect ReferencedSource { get; set; }
    public IfcText Description { get; set; }
    public IfcIdentifier Sort { get; set; }

    // === INVERSE CLAUSES ===
    // ClassificationRefForObjects : SET [0:?] OF IfcRelAssociatesClassification FOR RelatingClassification
    // HasReferences : SET [0:?] OF IfcClassificationReference FOR ReferencedSource
}
